README.md

```diff
@@ -14,9 +14,9 @@ sudo wget -O /databricks/jars/spark-video-assembly-0.0.2.jar https://github.com/
 ## For Developer
 ### Cheatsheet
 ```
-bin/mill video.test
-bin/mill video.publishLocal
-bin/mill video.assembly
+bin/mill 'video[2.12].test'
+bin/mill 'video[2.12].publishLocal'
```

Comment:
What's the generated jar name of `publishLocal` here, will it be with a `_2.12` suffix?

Reply:
Fixed now.
```
➜  spark-video git:(feature/cross_build) ✗ bin/mill 'video[2.12].publishLocal'
Compiling /Users/da/github/eto-ai/spark-video/build.sc
[40/48] video[2.12].docJar
model contains 23 documentable templates
[48/48] video[2.12].publishLocal
Publishing Artifact(eto.ai.rikai,spark-video_2.12,0.0.3-SNAPSHOT) to ivy repo /Users/da/.ivy2/local
➜  spark-video git:(feature/cross_build) ✗ ls ~/.ivy2/local/eto.ai.rikai/
spark-video  spark-video_2.12
➜  spark-video git:(feature/cross_build) ✗ bin/mill 'video[2.13].publishLocal'
[48/48] video[2.13].publishLocal
Publishing Artifact(eto.ai.rikai,spark-video_2.13,0.0.3-SNAPSHOT) to ivy repo /Users/da/.ivy2/local
➜  spark-video git:(feature/cross_build) ✗ ls ~/.ivy2/local/eto.ai.rikai/
spark-video  spark-video_2.12  spark-video_2.13
```